src/html.sortable.js

```diff
@@ -290,6 +290,7 @@ var _enableSortable = function (sortableElement) {
   var items = _filter(_getChildren(sortableElement), opts.items)
   var handles = _getHandles(items, opts.handle)
   _attr(sortableElement, 'aria-dropeffect', 'move')
+  _data(sortableElement, '_disabled', 'false')
```

Comment:
General fear and lack of understanding when it comes to JavaScript crazy type coercion, basically.

Also I didn't look too closely into where _data is stored, the _attr stuff appeared to always be strings.